src/cli/repo-checkout.js

```diff
@@ -102,11 +108,6 @@ export default async function main(config, argv) {
   // Checkout the underlying repository before running repo...
   await checkout(config, checkoutArgs);
 
-  // Running repo sync in parallel is only safe for entirely new initializations
-  // when we can though this is up to 2x faster...
-  let canUseParallelSync =
-    !await fs.exists(fsPath.join(args.directory, '.repo'));
```

Comment:
From IRC -- just trying to reduce un-tracked parallelism, this only applied to worker "first time" cache population.